iquip/apps/scheduler.py

```diff
@@ -106,11 +104,8 @@ class ExperimentView(QWidget):
     
     Attributes:
         experimentInfo: The ExperimentInfo instance that holds the experiment information.
-        layout: The list of ExperimentView widget.
-        args: The list of QLabel widget displaying the experiment information.
-        argsLayout: The HBoxLayout for displaying the experiment information besides its name.
+        argsLayout: The HBoxLayout for displaying the experiment information including its name.
```

Comment:
If `argsLayout` is just the layout of this widget, you can use [`layout()`](https://doc.qt.io/qt-5/qwidget.html#layout).
Is there a reason you made this as a public attribute?

Reply:
Hmm, I just followed `explorer.py`'s convention.

For now, I did not use it outside `ExperimentView` so I will make it local.

Maybe it can be used in the menu feature to detect the clicked region. I will add that again later if I need it.

Reply:
Oh which part of `explorer.py`? I don't see any public attribute for a layout.

Reply:
I think I saw something in my dream... Sorry 😂